### ISSUE TITLE
backup-list using the same environment variables works with wal-e but not with wal-g

### ISSUE THREAD

**Issue by avinashvallarapu** [wal-g/wal-g#485]
Hi,

I am trying to push backups using WAL-g now to Object Store (SWIFT). Following are the environment variables i am using. 

export SWIFT_TENANT=" "
export SWIFT_REGION=...
export SWIFT_AUTHURL=...
export WALG_SWIFT_PREFIX=...
export SWIFT_USER=...
export SWIFT_PASSWORD=...
export SWIFT_ENDPOINT_TYPE=...
export TMPDIR=...
export SWIFT_AUTH_VERSION="2.0"

It works perfectly well when i use **wal-e backup-list**. But i am getting the following error when i do the same with **wal-g**. Why is that ? Are there any other environment variables that need to be set ? 

$ wal-g backup-list
ERROR: 2019/12/03 14:27:25.772762 Swift error : Unable to authenticate connection: Can't find AuthVersion in AuthUrl - set explicitly

$ wal-g backup-push /pgdata/9.6
ERROR: 2019/12/03 14:52:36.424762 failed to configure folder: Swift error : Unable to authenticate connection: Can't find AuthVersion in AuthUrl - set explicitly

**Comment by x4m** [comment]
Hi! As far as I remember @tamalsaha and @iamrz1 are the best experts on swift support. Guys, what do you think about this problem?

**Comment by avinashvallarapu** [comment]
Hi Andrey,

I was reading through the Source and understood that the following variables need to be set. And it is working now. Thank You :) May be someone wants to confirm, i am doing further testing anyways, and considering to blog about it. 

WALG_SWIFT_PREFIX=..
OS_AUTH_URL=..
OS_USERNAME=..
OS_PASSWORD=..
OS_TENANT_NAME=" "
OS_REGION_NAME=...

**Comment by tamalsaha** [comment]
@avinashvallarapu , you are right. The constants are defined here: https://github.com/wal-g/wal-g/blob/master/internal/config.go#L91-L97

If you could, please send a pr to update the readme.

**Comment by iamrz1** [comment]
Hi @avinashvallarapu,

As you've correctly figured out, you have to set the prefix and standard v1, v2,or v3 [auth credentials for swift](https://www.swiftstack.com/docs/cookbooks/swift_usage/auth.html), as mentioned in the [readme](https://github.com/wal-g/wal-g/blob/master/README.md#L64-L66) file. If you feel the readme file needs to be more elaborate about prefix and credentials, please send in a PR with suggested changes. 
And do share with us if you write a blog about it. 

Thank you.
